components/sidebar/add_channel_dropdown/add_channel_dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+type Props = {
+    showMoreChannelsModal: () => void;
+    showNewChannelModal: () => void;
+};
+
+type State = {
+
+};
+
+export default class AddChannelDropdown extends React.PureComponent<Props, State> {
+    render() {
+        return (
+            <div>
+                <div>
+                    <a
+                        href='#'
+                        onClick={this.props.showMoreChannelsModal}
+                    >
+                        {'Browse Channels'}
```

Comment:
Can you use `FormattedMessage` for these and add them to `en.json` if they aren't already there?